frontend/src/metabase/common/hooks/use-collections-query/use-collections-query.ts

```diff
@@ -0,0 +1,20 @@
+import type { Collection, CollectionListQuery } from "metabase-types/api";
+import Collections from "metabase/entities/collections";
+
+import {
+  UseEntityListQueryProps,
+  UseEntityListQueryResult,
+  useEntityListQuery,
+} from "../use-entity-list-query";
+
+export const useCollectionsQuery = (
```

Comment:
`useCollectionListQuery` - please follow the established conventions

Reply:
3b8b52fc54755af96433335406bb470d8132fe94